sorc/ncep_post.fd/MISCLN.f

```diff
@@ -282,9 +297,15 @@ SUBROUTINE MISCLN
 
          DEPTH = 6000.0
          CALL CALHEL(DEPTH,UST,VST,HELI,USHR1,VSHR1,USHR6,VSHR6)
-
-         IF(IGET(430) > 0) THEN
+! 0-6 km shear magnitude
 !$omp parallel do private(i,j,jj)
```

Comment:
@EdwardColon-NOAA Please remove "jj" in line 301 which is not used in openmp code.